homeassistant/components/lyric/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*	@shellster
```

Comment:
I can't seem to get the hassfest script to run properly.  I am testing via the docker version, I think I need to run the script from within the running docker image.  I'll see if I can get that figured out.  

With regard to that other PR, I don't think it is very relevant anymore.  My version supports HEAT_COOL mode as well as heat and cool.  It supports fan modes as well.  The old one did none of these things, and more importantly, it was before the Climate redesign.  So, while I'm happy to consider anything relevant, I don't think much of that work is useful in this context.

As the linked PR points out the other Honeywell code base is for an entirely different website and series of tools.  Considering that the T-Series and Lyric thermostats have never had proper support, I would really like to get this PR to a point where it can be merged and get it merged so people with these thermostats have proper support.  After that, perhaps I can contact the developers of the other Honeywell package, and work on trying to unify this support for a later date.

It should be noted that this is my first contribution to this project, and I don't really know what I'm doing, I just have one of these thermostats and know a fair amount of python and want to use home-assistant.  So, please feel free to give me suggestions and or guidance to help bring this thing home.

Reply:
The linked PR was updated to be compatible with the new climate entity integration.

Reply:
I ran the script.hassfest, and committed the updated CODEOWNERS file, but it is still failing on that check, so I don't know what I'm doing wrong, there.

Reply:
I suspect that I'll have to pull the latest version of home-assistant and remerge the changes once the pypi library stuff gets updated and I fix that dependency.

Reply:
One last note (sorry for the comment spam).  I will be traveling for the next three weeks and will likely not be able to make changes or test until I get back.  However, if in that time there is no progress merging my changes upstream into python-lyric, I propose that I create an alternative package and integrate that when I get back.

Reply:
Rebase on latest dev branch then run the hassfest script again.